mypy.ini

```diff
@@ -49,3 +49,6 @@ ignore_missing_imports = True
 [mypy-urllib3]
 ; https://github.com/urllib3/urllib3/issues/867
 ignore_missing_imports = True
+
+[mypy-rfc3986]
+ignore_missing_imports = True
```

Comment:
👍 

Nitpick: I'd move this above the `mypy-setuptools` section, to keep things alphabetized.

(And maybe open an issue on `rfc3986` to see if they're interested in adding type annotations 😉)

Reply:
I have moved the missing imprt above `mypy-setuptools`, but I will wait for this PR to get approved so that we are comfortable with using `rfc3986` , and then open a issue and add it to here in the last commit before merge